app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1911,11 +1909,33 @@ BrowserWorld::CreateSkyBox(const std::string& aBasePath, const std::string& aExt
 
   if (extension == ".ktx") {
 #if defined(OPENXR) && defined(OCULUSVR)
```

Comment:
I think Lynx also supports SRGB; I understand that such a change doesn't make sense as part of this patch, but I'd rather integrate this "default format" logic in the GetAvailableColorFormat method, where we could have some extensible device specific logic.

Reply:
> I think Lynx also supports SRGB; I understand that such a change doesn't make sense as part of this patch.

Feel free to add this logic after this PR has been merged. I don't have a Lynx device, so I can't help test it.